animation_nodes/nodes/material/material_output.py

```diff
@@ -16,7 +16,7 @@
 
 class MaterialOutputNode(bpy.types.Node, AnimationNode):
     bl_idname = "an_MaterialOutputNode"
-    bl_label = "Material Output"
+    bl_label = "Workbench Material Output"
```

Comment:
Why should we rename it to Workbench? It doesn't seem to be restricted to it.

Reply:
As far as I know, the standard viewport renderer is also Workbench, so the node only affects Workbench.